packages/sdk/src/utils/discovery.ts

```diff
@@ -16,8 +20,14 @@ const DEFAULT_NODE_REQUIREMENTS = {
 export function defaultPeerDiscoveries(
   pubsubTopics: PubsubTopic[]
 ): ((components: Libp2pComponents) => PeerDiscovery)[] {
+  // TODO: add a check to see if it is indeed TWN or if it is a custom network
```

Comment:
nit: I think we don't need this comment and let's continue discussion in the issue